test/config/test_configgenerator.cc

```diff
@@ -11,10 +11,10 @@ using namespace ::testing;
 class ConfigGeneratorTest : public ::testing::Test {
 
 public:
-    ConfigGeneratorTest() {};
-    virtual ~ConfigGeneratorTest() {};
+    ConfigGeneratorTest() = default;;
```

Comment:
I suppose the second `;` is unwanted and happened by replacing the braces with `default`.

Reply:
Sounds about right. The ; was originally not necessary anyway.